tx-source/site_core.php

```diff
@@ -1696,12 +1696,75 @@
+$Definition['RichEditor.QuoteEnable.Notes'] = 'Use the following option to enable quotes for the Rich Editor. This will only apply if the default formatter is "Rich".';
+
+// Long Descriptions
+$Definition['richEditor.description.title'] = 'Rich Text Editor.';
+$Definition['richEditor.description.paragraphMenu'] = 'To edit a paragraph\'s style, hit tab to get to the paragraph menu. From there you will be able to pick one style. Nothing defaults to paragraph.';
+$Definition['richEditor.description.inlineMenu'] = 'An inline formatting menu will show up when you select text. Hit tab to get into that menu.';
+$Definition['richEditor.description.embed'] = 'Some elements, such as rich link embeds, images, loading indicators, and error messages may get inserted into the editor. You may navigate to these using the arrow keys inside of the editor and delete them with the delete or backspace key.';
+$Definition['richEditor.externalEmbed.description'] = 'This is an embed external element. It can be deleted using the delete key or the backspace key. To view the full element, press the preview button below.';
+
+// Formatting Menus
+$Definition['Inline Level Formatting Menu'] = 'Inline Level Formatting Menu';
+$Definition['Format as Bold'] = 'Format as Bold';
+$Definition['Bold'] = 'Bold';
+$Definition['Format as Italic'] = 'Format as Italic';
+$Definition['Italic'] = 'Italic';
+$Definition['Format as Strikethrough'] = 'Format as Strikethrough';
+$Definition['Strikethrough'] = 'Strikethrough';
+$Definition['Format as Inline Code'] = 'Format as Inline Code';
+$Definition['Paragraph Code Block'] = 'Paragraph Code Block';
+$Definition['Format as Link'] = 'Format as Link';
+$Definition['Link'] = 'Link';
+$Definition['Insert Url'] = 'Insert Url';
+$Definition['Paste or type a link…'] = 'Paste or type a link…';
+$Definition['×'] = '×';
+$Definition['Close'] = 'Close';
+$Definition['Line Level Formatting Menu'] = 'Line Level Formatting Menu';
+$Definition['Paragraph'] = 'Paragraph';
+$Definition['Format as Paragraph'] = 'Format as Paragraph';
+$Definition['Subtitle'] = 'Subtitle';
+$Definition['Format as Title'] = 'Format as Title';
+$Definition['Sub Subtitle'] = 'Sub Subtitle';
+$Definition['Format as Subtitle'] = 'Format as Subtitle';
+$Definition['Quote'] = 'Quote';
+$Definition['Format as blockquote'] = 'Format as blockquote';
+$Definition['Format as code block'] = 'Format as code block';
+$Definition['Spoiler'] = 'Spoiler';
+$Definition['Format as spoiler'] = 'Format as spoiler';
+$Definition['Inline Menu Available'] = 'Inline Menu Available';
+
+// Emoji Picker
+$Definition['Emoji'] = 'Emoji';
+$Definition['Emojis'] = 'Emojis';
+$Definition['Insert an emoji in your message.'] = 'Insert an emoji in your message.';
+$Definition['Use keyboard shortcuts "page up" and "page down" to cycle through available categories when menu is open.'] = 'Use keyboard shortcuts "page up" and "page down" to cycle through available categories when menu is open.';
```

Comment:
The key for this should be much shorter. Something like `'Use page up and page down to jump to categories.'` The translation can be the same.

Reply:
I feel this confuses things a bit more. A short string key like `richeditor.emojiPicker.pagingInstructions` would be more appropriate if we were to shorten the key.

Reply:
https://github.com/vanilla/vanilla/pull/7700